src/Components/Aspire.MySqlConnector/AspireMySqlConnectorExtensions.cs

```diff
@@ -0,0 +1,141 @@
+// Licensed to the .NET Foundation under one or more agreements.
+// The .NET Foundation licenses this file to you under the MIT license.
+
+using System.Data.Common;
+using Aspire;
+using Aspire.MySqlConnector;
+using HealthChecks.MySql;
+using Microsoft.Extensions.Configuration;
+using Microsoft.Extensions.DependencyInjection;
+using Microsoft.Extensions.Diagnostics.HealthChecks;
+using Microsoft.Extensions.Logging;
+using MySqlConnector;
+
+namespace Microsoft.Extensions.Hosting;
+
+/// <summary>
+/// Extension methods for connecting MySQL database with MySqlConnector client
+/// </summary>
+public static class AspireMySqlConnectorExtensions
+{
+    private const string DefaultConfigSectionName = "Aspire:MySql";
```

Comment:
Following our existing patterns, this should be

```suggestion
    private const string DefaultConfigSectionName = "Aspire:MySqlConnector";
```